meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,134 @@
+
+For more information on the main improvements, check out the following issues: meilisearch/product#513 and meilisearch/product#520.
+
+Made by @loiclec and @irevoire in #2689, #2690, meilisearch/milli#607, meilisearch/milli#606, meilisearch/milli#587, meilisearch/milli#557,meilisearch/milli#579, and meilisearch/milli#590.
+
+## Filtering: introduce the `EXISTS` and `IN` operators
+
+You can now use the `EXISTS` and `IN` operators with the `filter` search parameter during [search](https://docs.meilisearch.com/reference/api/search.html#search-in-an-index-with-post-route).
+
+With the following set of documents:
+
+```json
+{ "id": 1, "product": "T-shirt", "price": 20, "color": "yellow" }
+{ "id": 2, "product": "T-shirt",              "color": "red" }
+{ "id": 3, "product": "T-shirt", "price": 30 }
+{ "id": 4, "product": "T-shirt", "price": 40 }
+```
+
+- The filter `price EXISTS` will select documents `1`, `3`, and `4`.
+- The filter `price IN [30, 20]` will select documents `1` and `3`.
+
+These operators can be combined with the existing `NOT` (`!=`) operator.
+More about [filtering](https://docs.meilisearch.com/learn/getting_started/filtering_and_sorting.html).
+
+Made by @loiclec in #2636, meilisearch/milli#596, and meilisearch/milli#556.
+
+## Key management
+
+When managing [keys](https://docs.meilisearch.com/learn/security/master_api_keys.html#protecting-a-meilisearch-instance), the `*` helper can now be used to express a set of actions in the `actions` array.
+
+For example, instead of specifying `indexes.create`, `indexes.get`, `indexes.update`, `indexes.delete`; you can now use `indexes.*`.
+
+Made by our dear contributors @Mcdostone and @phdavis1027 in #2080 and #2579.
+
+# Behavior changes ⚠️
+
+## Auto-batching
+
+This previously experimental feature is now activated by default.
+
+For people who activated it in previous versions using `--enable-auto-batching`, you need to remove this flag when running Meilisearch. All the previous settings (`--debounce-duration-sec`, `--max-batch-size` and `--max-documents-per-batch`) have been removed.
+
+For people who hadn't activated it, it does not change anything but increases the indexing speed 🏎️
+
+If there are any issues during indexing, the `--disable-auto-batching` flag (or the environment variable `MEILI_DISABLE_AUTO_BATCHING`) is available.
+
+Made by @Kerollmops in meilisearch/milli#561, #2523, and #2677.
+
+## `!=` (`NOT`) filter operator
+
+This behavior change is consistent with the `EXISTS` and `IN` operators. The `NOT` filter now also matches documents that don't have the related attributes.
+More explanation with examples [here](https://github.com/meilisearch/meilisearch/issues/2486).
+
+Made by @loiclec in #2636, and meilisearch/milli#596.
+
+# Fixes 🐞
+
+* Clamp databases max size to a multiple of system page size (#2662) @ManyTheFish
+* Prevent removing whitespace trimming during document id validation (#2636 and meilisearch/milli#599) @ManyTheFish
```

Comment:
I think there's an extra verb here? Otherwise I'm not sure I understand this one.

```suggestion
* Prevent whitespace trimming during document id validation (#2636 and meilisearch/milli#599) @ManyTheFish
```